src/nixpacks/builder/docker/dockerfile_generation.rs

```diff
@@ -232,7 +232,7 @@ impl DockerfileGenerator for StartPhase {
     ) -> Result<String> {
         let start_cmd = match &self.cmd {
             Some(cmd) => utils::get_exec_command(cmd),
-            None => "".to_string(),
+            None => bail!("No start command cound be found"),
```

Comment:
The dockerfile generation should not error. It should error when the build plan is created/validated